src/Akeneo/UserManagement/back/ServiceApi/User/UsersQuery.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Akeneo PIM Community Edition.
+ *
+ * (c) 2022 Akeneo SAS (https://www.akeneo.com)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Akeneo\UserManagement\ServiceApi\User;
+
+use Akeneo\UserManagement\Domain\Storage\FindUsers;
+
+final class UsersQuery
+{
+    public function __construct(
+        private ?string $searchName = null,
```

Comment:
```suggestion
        private ?string $search = null,
```
As we also search in email, username, etc... `$search` is more appropriate I think.